meteor/client/ui/SegmentTimeline/Renderers/VTSourceRenderer.tsx

```diff
@@ -484,11 +405,17 @@ export class VTSourceRendererBase extends CustomLayerItemRenderer<IProps & WithT
 			<span className="segment-timeline__piece__label" ref={this.setLeftLabelRef} style={this.getItemLabelOffsetLeft()}>
 				{noticeLevel !== null && <PieceStatusIcon noticeLevel={noticeLevel} />}
 				<span
-					className={ClassNames('segment-timeline__piece__label', {
-						'overflow-label': end !== '',
-					})}
+					className={ClassNames(
+						'segment-timeline__piece__label',
+						'with-duration',
```

Comment:
Same problem as in https://github.com/nrkno/sofie-core/blob/05a90536a680bc75ca10ad950a53f2ae2ce9ac01/meteor/client/ui/SegmentTimeline/Renderers/DefaultLayerItemRenderer.tsx#L59 - the `with-duration` class is added, even though `this.renderDuration()` makes an independent decision if it will render the duration or not.

Reply:
@jstarpl So the idea is that this class does not decide if the duration should be shown or not and it does not break the label if the duration is not shown later on. It just means that there might be a duration displayed in that label